app/views/dashboard/_status_widget.html.erb

```diff
@@ -23,7 +23,7 @@
   %>
 
   <%= searchable_links _('Out of sync hosts'),
```

Comment:
Though the calculation is perhaps for the same thing - to only consider hosts that are in sync and some other state?